README.md

```diff
@@ -13,25 +44,77 @@ git clone https://github.com/klavenj/open-forum-backend.git
 Go into the project directory
 
 ```bash
+
 cd open-forum-backend/
 ```
 
 Checkout working branch
 
 ```bash
+
 git checkout -b <branch>
 ```
 
 Install NPM dependencies
 
 ```bash
+
 npm i
 ```
 ---
 
-## Environment Variables
 
-For local developmebt, You can duplicate .env.example located file and rename it to .development.env or .env
+
+# Environment Set Up
+
+
+## Set up Database
+
+### Start Postgres (If not using the Postgres.app)
+
+```bash
+
+brew services start postgres
```

Comment:
You typically don't need specific instructions for setting up a database server (which would be different on Linux or Windows anyway).